You would get exceptions when you try to call a class method on an instance (`tv.manufacter`) and when you try to call an instance method on the class (`Television.model`). The other method calls would behave as expected.